### PR TITLE
Add ChosenInlineResult handler

### PR DESCRIPTION
**Pull request by rojvv**
# What

`ChosenInlineResult` update handler.

# Impact

- Are your changes backwards compatible? Yes
- Have you included documentation, or updated existing documentation? -
- Do errors and log messages provide enough context? -

### CODE REVIEW

File: ext/handlers/filters/choseninlineresult.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	}
+}
+
+func ChosenResultQuery(q string) ChosenInlineResult {
+	return func(cir *gotgbot.ChosenInlineResult) bool {
+		return cir.Query == q
+	}
+}
+
+func ChosenResultPrefix(prefix string) ChosenInlineResult {
```
Comment (PaulSonOfLars) [inline ext/handlers/filters/choseninlineresult.go]: Would it be better to call this `ChosenResultQueryPrefix` instead? Would make it easier to read the code and know what the prefix is for. What do you think?

**Review by PaulSonOfLars — COMMENTED** [review]
Thank you for the contribution! 

Got one question regarding the naming of the prefix/suffix methods that id like your opinion on, otherwise looks great :)

File: ext/handlers/filters/choseninlineresult.go (hunk 1 above)
Comment (PaulSonOfLars) [reply ext/handlers/filters/choseninlineresult.go]: (of course, i see this isnt the case for the current callback query filter; If you agree with the naming scheme, I'll rename that in a later commit)

Comment (rojvv) [reply ext/handlers/filters/choseninlineresult.go]: Done!

**Pull request merged**